### ISSUE TITLE
TargetServer requires user input to start

### ISSUE THREAD

**Issue by andrewgaul** [disy/jSCSI#21]
Presently TargetServer emits:

```
This system provides more than one IP Address to advertise.

[0] 172.17.42.1
[1] fe80:0:0:0:6676:baff:fe92:27c6%wlan0
[2] 192.168.1.73
[3] 0:0:0:0:0:0:0:1%lo
[4] 127.0.0.1
```

and requires user input to proceed.  Instead it should read this value from the config file.

**Comment by sebastiangraf** [comment]
you're right. Unfortunately I have to admin that this project is here for legacy only. I left the disy-group and have no ressourcen to maintain this project actively.
I will leave this issue open until I or someone else finds time to fix it.

Thanks for the other PullRequests. I really appreciate that:)

**Comment by ActiveVolcano** [comment]
IP Address can be specified in start code like

```java
import org.jscsi.target.Configuration;
import org.jscsi.target.TargetServer;

var config = Configuration.create (new File ("jscsi-target.xsd"), new File ("jscsi-target.xml"), "127.0.0.1");
new TargetServer (config).call();
```

**Comment by ActiveVolcano** [comment]
Besides the way of passing target address as Configuration class constructor parameter mentioned above,
here is a pull request #45 which loads target address setting from XML file.
No user input required anymore.
